app/controllers/organen/orgaan/index.js

```diff
@@ -1,15 +1,21 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
 
-export default class OrganenOrgaanController extends Controller {
-  queryParams = ['bestuursperiode'];
+export default class OrganenOrgaanIndexController extends Controller {
+  @service router;
+  @tracked isModalActive = false;
 
-  @tracked bestuursperiode;
+  @action
+  toggleModal() {
+    this.isModalActive = !this.isModalActive;
+  }
 
   @action
-  selectPeriod(period) {
-    this.bestuursperiode = period.id;
+  onSave() {
+    this.isModalActive = !this.isModalActive;
+    setTimeout(() => this.router.refresh(), 1000);
```

Comment:
Hm why are we doing this ?

Reply:
That is an issue with caching, since we use the form-content service and not the resources directly, the frontend does not know when our resource has actually been updated so it can't refresh at the appropriate time. That's why we use an arbitrary timeout, we should fix this at some point, but I think it was difficult to do.